fsharp-backend/src/ClientTypes/Converters.fs

```diff
@@ -0,0 +1,19 @@
+module ClientTypes.Converters
+
+module STJ =
```

Comment:
Perhaps there's an opportunity to get rid of this. We could argue that the ClientType here is really a `string` and the conversion functions should do the parsing, rather than rely on this (I feel this particular thing is annoying to learn and keep track of)

Reply:
I don't understand what you mean here. Could you please rephrase or briefly chat?

Reply:
Never mind - I understand now.